.github/workflows/e2e.yml

```diff
@@ -0,0 +1,67 @@
+name: End to End Tests
+on: 
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest]
+        vscodeVersion:
+          - stable
+    runs-on: ${{ matrix.os }}
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            path: ./salesforcedx-vscode
+            ref: ${{ github.event.ref }}
```

Comment:
This might be shorter too now, but since it's not all that verbose compared to the inputs I don't hate it as is.

Reply:
might not even need it as this might be the default value.  also don't hate just having it.

Reply:
So once we get the workflow merged in this is what will let us run against specific branches, so this one is useful for us. Right now that's all manual, but eventually we could be have it be kicked off from a pull request or another event.

Reply:
Confirmed this syntax looks ok for us: https://github.com/randi274/salesforcedx-vscode-fork/actions/runs/4800025121/jobs/8540419374